### PR TITLE
fix all those issues I just made

### LINKED ISSUE #193: allow disabling stat icons
stat icons are *very* slow. it should be possible to disable them, and perhaps we should implement a cap on top-level classes before they're automatically disabled

### LINKED ISSUE #194: always show dev menu if a development option is enabled

### PR DESCRIPTION
**Pull request by ix0rai**
fix #193 
fix #194 
will make attempts at #192

### CODE REVIEW

File: enigma-swing/src/main/java/org/quiltmc/enigma/gui/config/DevSection.java [hunk 1]
```diff
@@ -1,21 +1,20 @@
 package org.quiltmc.enigma.gui.config;
 
-import com.google.gson.annotations.SerializedName;
```
Comment (IotaBread) [inline enigma-swing/src/main/java/org/quiltmc/enigma/gui/config/DevSection.java:3]: :p

**Comment by ix0rai** [comment]
@IotaBread imo we merge this unfinished for 2.3 along with the ASM fix and then address the last issue in the next update

**Comment by IotaBread** [comment]
sounds good to me!

**Comment by ix0rai** [comment]
gorgeous let's do it

**Pull request merged**